crates/witx2/tests/all.rs

```diff
@@ -100,19 +100,7 @@ impl Runner<'_> {
         let result = if contents.contains("// parse-fail") {
             match result {
                 Ok(_) => bail!("expected test to not parse but it did"),
-                Err(e) => {
-                    if cfg!(windows) {
-                        // Normalize path separator and system error on Windows
-                        format!("{:?}", e)
-                            .replace("\\parse-fail\\", "/parse-fail/")
-                            .replace(
```

Comment:
I think this replacement is still needed given the difference in system error messages.